tskv/src/reader/column_group/mod.rs

```diff
@@ -75,7 +75,7 @@ impl ColumnGroupReader {
         })
     }
 
-    pub fn new_with_unchecked(
+    pub fn _new_with_unchecked(
```

Comment:
rm this

Reply:
This function is used in the test module.